### PR TITLE
Requirements Update 2018-01-02

### PR DESCRIPTION
**Pull request by encbladexp**
Some small updates on our requirements.

### CODE REVIEW

File: xknx/knx/dpt_hvac_mode.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ def from_knx(cls, raw):
         elif raw[0] == 0x00:
             return HVACOperationMode.AUTO
 
+        return None
```
Comment (Julius2342) [inline xknx/knx/dpt_hvac_mode.py]: Shouldn't we throw an exception here if we can't parse the KNX value? e.g. `raise CouldNotParseKNXIP("Could not parse HVACOperationMode")`

File: xknx/knx/address_filter.py [hunk 2]
```diff
@@ -48,13 +48,16 @@ def match(self, address):
...
             address = Address(address)
+
         if len(self.level_filters) == 3:
             return self._match_level3(address)
         elif len(self.level_filters) == 2:
             return self._match_level2(address)
         elif len(self.level_filters) == 1:
             return self._match_free(address)
 
+        return None
```
Comment (Julius2342) [inline xknx/knx/address_filter.py]: Why not a `return false` bc we can't match ...

File: xknx/knx/dpt_hvac_mode.py [hunk 3]
```diff
@@ -76,6 +80,8 @@ def from_knx(cls, raw):
         elif raw[0] & 1 > 0:
             return HVACOperationMode.COMFORT
 
+        return None
```
Comment (Julius2342) [inline xknx/knx/dpt_hvac_mode.py]: same here, why not raise an Exception ...

**Comment by encbladexp** [comment]
I updated this PR.

**Pull request merged**